tests/regression/01-cpa/71-widen-sides.c

```diff
@@ -0,0 +1,28 @@
+// PARAM: --set ana.ctx_insens "['base', 'mallocWrapper']"  --enable ana.int.interval --sets solvers.td3.side_widen sides-local
+int further(int n) {
```

Comment:
Required after #845:
```suggestion
#include <goblint.h>

int further(int n) {
```